src/legacy/test-run/index.js

```diff
@@ -60,8 +51,7 @@ export default class TestRun extends Session {
             stepNames:              JSON.stringify(this.test.stepData.names),
             testSteps:              this.test.stepData.js,
             sharedJs:               sharedJs,
-            nextStep:               nextStep,
-            deferredError:          this.deferredError ? JSON.stringify(this.deferredError) : 'null',
+            sessionId:              this.id,
```

Comment:
@inikulin do you mean we can get it from `hammerhead` on the client instead of sending from the server? If so, I don't see this settings in the `hammerhead` [client api](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/index.js)